<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>560</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>RenobeParser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icon/icons8-my-melody-480.png</normaloff>:/icon/icons8-my-melody-480.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
	background:  #272727;
}
</string>
  </property>
  <widget class="QListView" name="recomendList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>780</width>
     <height>380</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#recomendList{
	background: #48494a;
}</string>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QListView" name="searchResult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>66</y>
     <width>675</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#searchResult{
	background: #48494a;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="searchEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>675</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#searchEdit{
	background: #000000;
	color: #ffffff;
	font-size: 18px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>30</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#searchButton{
	background-color: #000000;
	font-family: monospace;
	color: #ffffff;
}

#searchButton:hover{
	background-color: #666;
	color: #272727;
}

</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
